lib/mix/lib/mix/tasks/new.ex

```diff
@@ -126,7 +126,7 @@ defmodule Mix.Tasks.New do
   defp generate_umbrella(_app, mod, path, _opts) do
     assigns = [app: nil, mod: mod]
 
-    create_file ".gitignore", gitignore_text()
+    create_file ".gitignore", gitignore_text(assigns)
```

Comment:
As you can see the build fails. If you use assigns, you should use a template not a text (`embed_template`). Always make sure to run `make test` to check everything's alright :)